dimelo/parse_bam.py

```diff
@@ -538,7 +550,7 @@ def verify_inputs(
 
 def check_bam_format(
     bam_file: str | Path,
-    basemods: list = ["A,0", "CG,0"],
+    motifs: list = ["A,0", "CG,0"],
```

Comment:
Question: have we changed `basemods` to `motifs` everywhere in the package? Seems like a good idea to me. If not, can I request you to make that change?

Reply:
In my heart we have. In reality, I haven't checked.